packages/hub/services/json-middleware.ts

```diff
@@ -0,0 +1,73 @@
+import compose from 'koa-compose';
+import route from 'koa-better-route';
+import Koa from 'koa';
+// @ts-ignore
+import mimeMatch from 'mime-match';
+import KoaBody from 'koa-body';
+import { Memoize } from 'typescript-memoize';
+import { CardstackError } from '../utils/error';
+import { SessionContext } from './authentication-middleware';
+import { inject } from '../di/dependency-injection';
+import WyreCallbackRoute from '../routes/wyre-callback';
+
+const ROUTE_PREFIX = '/callbacks';
```

Comment:
`callbacks-middleware` or `webhooks-middleware` would be a more descriptive name for this service.

Reply:
Sounds good.